sdk/aqueduct/backend/api_client.py

```diff
@@ -537,13 +537,17 @@ def get_artifact_result_data(
         parsed_response = _parse_artifact_result_response(resp)
         execution_status = parsed_response["metadata"]["exec_state"]["status"]
 
-        if execution_status != ExecutionStatus.SUCCEEDED:
-            print("Artifact result unavailable due to unsuccessful execution.")
-            return None, execution_status
-
         serialization_type = parsed_response["metadata"]["serialization_type"]
         artifact_type = parsed_response["metadata"]["artifact_type"]
-        return (
-            deserialize(serialization_type, artifact_type, parsed_response["data"]),
-            execution_status,
-        )
+
+        return_value = None
+        if "data" in parsed_response:
+            return_value = deserialize(serialization_type, artifact_type, parsed_response["data"])
+
+        if execution_status != ExecutionStatus.SUCCEEDED:
+            if return_value is not None:
+                print(f"Artifact execution failed with return value {return_value}")
```

Comment:
Hmm let's remove this print statement. The user already gets the value back, they don't need it printed for them.

Reply:
done